### PR TITLE
Update default Rules of cart promotion

### PR DESCRIPTION
**Pull request by zairigimad**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12 or 1.13                   |
| Bug fix?        | no                                                      |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | fixes #X, partially #Y, mentioned in #Z                      |
| License         | MIT                                                          |

### CODE REVIEW

**Review by Prometee — DISMISSED** [review]

File: docs/book/orders/cart-promotions.rst [hunk 1]
```diff
@@ -62,6 +62,8 @@ The types of rules that are configured in **Sylius** by default are:
 * **Items From Taxon Total** - checks in the cart if items from a given taxon cost a given amount of money,
 * **Nth Order** - checks if this is for example the second order made by the customer,
 * **Shipping Country** - checks if the order's shipping address is in a given country.
+* **Customer Group** - checks if the current customer is in a given customer group.
+* **Contain Product** - checks if the order contain a certain product.
```
Comment (NoResponseMate) [inline docs/book/orders/cart-promotions.rst]:
```suggestion
* **Contains Product** - checks if the order contains a certain product.
```

**Review by NoResponseMate — DISMISSED** [review]

File: docs/book/orders/cart-promotions.rst [hunk 2]
```diff
@@ -62,6 +62,8 @@ The types of rules that are configured in **Sylius** by default are:
 * **Items From Taxon Total** - checks in the cart if items from a given taxon cost a given amount of money,
 * **Nth Order** - checks if this is for example the second order made by the customer,
 * **Shipping Country** - checks if the order's shipping address is in a given country.
+* **Customer Group** - checks if the current customer is in a given customer group.
+* **Contains Product** - checks if the order contains a certain product.
```
Comment (diimpp) [inline docs/book/orders/cart-promotions.rst:66]: Would be cool to sort rules the same way as in the admin panel

image

Some of the rules no longer match admin panel (Not needed captial letters at each word for rest),
```patch
--- Taxon
--- Items From Taxon Total 
+++ Has at least one from taxons
+++ Total price of items from taxon
```

```
Cart quantity
Customer group
Nth order
Shipping country
Has at least one from taxons
Total price of items from taxon
Contains product
Item total
```

@zairigimad Hi, great catch :+1:  Please let me know if you would like to handle this, or I will do a follow up PR.

**Review by diimpp — DISMISSED** [review]

File: docs/book/orders/cart-promotions.rst (hunk 2 above)
Comment (zairigimad) [reply docs/book/orders/cart-promotions.rst:66]: Nice catch too @diimpp I can add it if you want in this PR since it’s not merged yet.

Comment (diimpp) [reply docs/book/orders/cart-promotions.rst:66]: That would be awesome, I'll raise this PR for faster merge afterwards.

Sorry for a long wait :pray:

Comment (zairigimad) [reply docs/book/orders/cart-promotions.rst:66]: it's done.

**Review by diimpp — APPROVED** [review]

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thank you, @zairigimad!

**Pull request merged**